public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -163,6 +181,31 @@ export function LokiContextUi(props: LokiContextUiProps) {
 
   return (
     <div className={styles.wrapper}>
+      <Button
+        title="Revert to initial log context query."
+        icon="history-alt"
+        variant="secondary"
+        className={cx(
+          styles.iconButton,
+          css`
+            visibility: ${isInitialQuery ? 'hidden' : 'visible'};
+          `
+        )}
+        tooltip="Revert to initial log context query."
+        onClick={(e) => {
+          e.stopPropagation();
+          setContextFilters((contextFilters) => {
```

Comment:
What do you think about adding feature tracking here?

Reply:
Yes, good idea. Added in https://github.com/grafana/grafana/pull/68484/commits/cd6c065bfea9d291d76c6798d9447188f38c4d57